src/styles/blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: $product-card-width;
+
+  border: 1px solid $product-border-color;
+  border-radius: $product-border-radius;
```

Comment:
Ensure that the SCSS variables `$product-border-color` and `$product-border-radius` are defined in your project. Undefined variables will cause a compilation error.